qa/L0_model_config/autofill_noplatform/tensorrt/bad_dynamic_shapes_max/expected

```diff
@@ -1 +1 @@
-model configuration specified invalid shape for input 'INPUT0' for 'bad_dynamic_shapes_max_0_gpu0'. Error details: model expected the shape of dimension 0 to be between 4 and 32 but received 33
+model configuration specified invalid shape for input 'INPUT0' for model bad_dynamic_shapes_max. Error details: model expected the shape of dimension 0 to be between 4 and 32 but received 33
```

Comment:
Why are we removing the single quotes around model name?

Reply:
The new TRT backend doesn't include the quotes around the names. It's a quick fix for CI for now. I will incorporate the quotes change sometime in the future once more important issues are resolved.

Reply:
Sounds good. Marking this as unresolved for now to ensure we don't miss this.